docs/language/ql-training/cpp/control-flow-cpp.rst

```diff
@@ -2,11 +2,7 @@
 Analyzing control flow
```

Comment:
Can't comment inline, but [line 92](https://github.com/Semmle/ql/pull/2257/files#diff-7c46c31059ce63c74c0f2febd77649b3R92) of this file should talk about the "(CodeQL) class" instead of "QL class".